core/src/main/kotlin/behavior/GlobalApplicationCommandBehavior.kt

```diff
@@ -52,6 +50,19 @@ interface GlobalApplicationCommandBehavior : ApplicationCommandBehavior {
     override suspend fun delete() {
         service.deleteGlobalApplicationCommand(applicationId, id)
     }
+
+    /**
+     * Updates the permissions for this command on the guild corresponding to [guildId].
+     *
+     * @throws [RestRequestException] when something goes wrong during the request.
+     */
+    suspend fun editPermissions(
```

Comment:
So you can even modify the permissions of global commands?

Reply:
yes you can modify perms of a global command on guilds